setup-qubes.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# exit on errors, undefined variables, ensure errors in pipes are not hidden
+set -Eeuo pipefail
+
+PREFIX_APP_VM="A-"
+PREFIX_TEMPLATE_VM="Z-"
+
+# fetchFromVM VMNAME FILE [EXE]
+function fetchFromVm() {
+	if [ $# -lt 2 ]; then
+		echo "Expected at least two parameters: fetchFromVm SOURCEVMNAME FILENAME [EXE]"
+		return 1
+	fi
+	VMNAME="${1}"
+	FILE="${2}"
+	EXE="${3}"
+
+	echo "Fetching ${FILE} from VM ${VMNAME}..."
+	
+	# delete file in case it already exists on dom0 and ignore errors
+	FILENAME=$(basename "$FILE")
+	rm $FILENAME 2>>/dev/null
+
+	# fetch the file via the 'cat' hack to avoid dom0 security precautions 
+	if qvm-run -p ${VMNAME} cat ${FILE} >> $FILENAME; then
```

Comment:
The `qvm-run` command needs to be installed. It's a good practice to create some kind of check function executed at the beginning of the script where to check if special command lines are installed or not.

Reply:
good point - will add this. This should be a given on QubesOS but maybe user is running this not in dom0 and then it should not just do undefined things.